MC/Geant4Config.C

```diff
@@ -38,6 +51,14 @@ void Geant4Config()
 	Printf("We cannot use the BERT+biasing with the monopoles, we will unset the pointer to geant4");
 	delete geant4;
 	geant4 = 0;
+	delete og4;
+	og4 = 0;
+	return;
+      }
+      else if (str->String().Atoi() < 4 || !g4Version.Contains("monopole")) {
+	Printf("We cannot use monopoles with the current version of G4: %s", str.Data());
```

Comment:
str.Data does not compile, after changing to str->String().Data() it seems to work on Root6

Reply:
and another minor fix to remove a warning in root6 
change
      Printf("\n PDG Code: %d, mass: %3.3f, el. charge: %d; mag. charge: %d",pdgConfig,mass,eCh,mCh);
to
      Printf("\n PDG Code: %d, mass: %3.3f, el. charge: %1.0f; mag. charge: %1.0f",pdgConfig,mass,eCh,mCh);

Reply:
About the str.Data(), you are right, sorry, my bad, I overlooked it. 
I will fix both things.

Chiara